File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/AssetReeferResponseReeferStatsEngineHours.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param engineHours Engine hours of the reefer.
 * @param changedAtMs Timestamp in Unix milliseconds since epoch.
 */
data class AssetReeferResponseReeferStatsEngineHours (

        @JsonProperty("engineHours") val engineHours: Long? = null,

        @JsonProperty("changedAtMs") val changedAtMs: Long? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/AssetReeferResponseReeferStatsEngineHours.scala

package org.openapitools.client.model


case class AssetReeferResponseReeferStatsEngineHours (
    /* Engine hours of the reefer. */
    _engineHours: Option[Long],
    /* Timestamp in Unix milliseconds since epoch. */
    _changedAtMs: Option[Long]
)
object AssetReeferResponseReeferStatsEngineHours {
    def toStringBody(var_engineHours: Object, var_changedAtMs: Object) =
        s"""
        | {
        | "engineHours":$var_engineHours,"changedAtMs":$var_changedAtMs
        | }
        """.stripMargin
}
